packages/web3-core/CHANGELOG.md

```diff
@@ -201,8 +201,10 @@ Documentation:
 
 -   Added `isMetaMaskProvider` function to check if provider is metamask (#6534)
 
-## [Unreleased]
+## [4.4.0]
```

Comment:
Its a patch fix for following change so should it be 4.3.2

Reply:
updated to 4.3.2